src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -1080,6 +1080,7 @@ private void JSON_response(HttpServletResponse response, String jsonValue,
 		try {
 			// output json
 			response.setContentType(MediaType.APPLICATION_JSON);
+			response.setCharacterEncoding("UTF-8");
```

Comment:
This is set for all the Json_response that get sent to the browser. The json response is what hold the data and it makes sense to make it set the encoding. Not all responses are set to UTF-8 encoding i think and for some it is not needed. Like for Zip download the response is a downloadable and i tried to set to UTF-8 encoding when the folder name have non-ascii value but folder name with non-ascii values downloaded correctly

I read/thought little bit about setting all response to uft-8 encoding like creating a filter before MigrationController or do something to put that in place. Not sure if we need that design in place. And i know @dlhaines  you will ask this question so i have opened it as s discussion point.